### ISSUE TITLE
Preview window with specific size

### ISSUE THREAD

**Issue by scofano** [aseprite/aseprite#1362]
It would be nice if we could type the preview window size.

So we could see what sprites would look like on an android device, while working on the details of the image with 400% zoom.

**Comment by dacap** [comment]
Hi @scofano, do you mean to change the window width x height size right? so we could have a sense about how much device space the image would use.

**Comment by scofano** [comment]
Yeah. I´m trying to develop iphone 6 mockups (for games), and I´d like to keep the preview window size fixed on iphone´s proportion but I can´t resize it using px sizes.

It would be something like this:
![aseprite](https://cloud.githubusercontent.com/assets/24534225/21181359/a786750a-c1e4-11e6-94d2-52566f5f523d.gif)

**Comment by scofano** [comment]
Well, I´ve been thinking about it and I dont think it is necessary to have numbers to input anymore.
Just showing the window size is enough. We can drag window borders until it reaches the desired size. :)

**Comment by MatthiasBae** [comment]
Hello, the request is quite old but I think it would be a great feature. 

I am working on a game which is 960x540 pixels. Each tile has 16pixels so my game has 60x34 tiles. Shall I now scroll in the preview window and count the tiles until it has got the correct preview of my game? This workflow is not optimal. 

Maybe you can rethink about it again :)
